front/src/modules/command-menu/states/queuedAction.tsx

```diff
@@ -0,0 +1,6 @@
+import { atom } from 'recoil';
```

Comment:
The file should be named as the atom, this is the current convention we use on the repo, so here it would be queuedActionsState.ts